src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/RazorHtmlCSharpLanguageServer.cs

```diff
@@ -293,6 +295,17 @@ public Task<IReadOnlyList<VSInternalDiagnosticReport>> DocumentPullDiagnosticsAs
             return Task.FromResult<VSInternalWorkspaceDiagnosticReport?>(null);
         }
 
+        // Workaround for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1445500
+        // The Web Tools WrapWithTag handler sends messages to all of the LSP servers attached to the buffer
+        // and we respond correctly from the RazorLanguageServer, but unless we implement a handler here, the
+        // platform will get a "Request Method Not Found" exception, and throw away the real result from the Razor
+        // server entirely.
+        [JsonRpcMethod(LanguageServerConstants.RazorWrapWithTagEndpoint, UseSingleObjectParameterDeserialization = true)]
+        public static Task<WrapWithTagResponse?> WrapWithTagAsync(WrapWithTagParams workspaceDiagnosticsParams, CancellationToken cancellationToken)
```

Comment:
At a later date it probably makes sense to add a server capability indicating that the server supports wrap with div too